src/ralph/data_importer/management/commands/importer.py

```diff
@@ -71,6 +71,12 @@ def add_arguments(self, parser):
             default='utf-8',
             help="Output encoding",
         )
+        parser.add_argument(
+            '-f', '--from_ralph',
```

Comment:
i'd rather name this argument `--use-imported-id` and use this as a boolean flag.